Extension/src/LanguageServer/configurations.ts

```diff
@@ -351,6 +351,9 @@ export class CppProperties {
             }
         } catch (error) {} finally {
             this.vcpkgPathReady = true;
+            if (this.vcpkgIncludes.length) {
+                telemetry.logLanguageServerEvent('vcpkgUserPlatform', { 'platform': await os.platform() });
```

Comment:
I don't believe this was on our telemetry list.  In any case the OS info is automatically appended to all events and it is unnecessary to add.